packages/fxa-react/components/LoadingSpinner/index.stories.tsx

```diff
@@ -1,7 +1,12 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import LoadingSpinner from './index';
+import LoadingSpinner, { SpinnerType } from './index';
 
-storiesOf('Components/LoadingSpinner', module).add('basic', () => (
-  <LoadingSpinner />
-));
+storiesOf('Components/LoadingSpinner', module)
+  .add('default', () => <LoadingSpinner />)
+  .add('blue', () => <LoadingSpinner spinnerType={SpinnerType.Blue} />)
+  .add('white', () => (
+    <div style={{ backgroundColor: 'gray' }}>
```

Comment:
Nit: Can we change this to `className="bg-grey-700"` since we've been converting to use Tailwind? 😅